docs/builtin-plugins.md

```
@@ -195,10 +195,10 @@ bot.add_plugin(TranslatePlugin())
 
 ### AI plugins
 
-**`OpenClaudePlugin`** (also exported as `AIPlugin`) — `/chat` — conversational AI powered by your configured provider. Requires `Bot(ai_provider=…)`.
+**`AIPlugin`** — `/ask` — conversational AI powered by your configured provider. Requires `Bot(ai_provider=…)`. `OpenClaudePlugin` is a backwards-compatible subclass of `AIPlugin` and can be used interchangeably.
 ```python
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)